src/meshable_arena.cc

```diff
@@ -350,7 +350,7 @@ void MeshableArena::partialScavenge() {
   forEachFree(_dirty, [&](const Span &span) {
     auto ptr = ptrFromOffset(span.offset);
     auto sz = span.byteLength();
-    madvise(ptr, sz, MADV_DONTNEED);
+    madvise(ptr, sz, MADV_DONTNEED|MADV_DONTDUMP);
```

Comment:
`MADV_DONTNEED` is a temporary flag to free the physical page associated with a mapping, and the OS will automatically allocate a new page on demand the next time we write to the page.  Does `DONTDUMP` work similarly, or will it exclude that page from coredumps until the program exits (or another `madvise` call removes the flag)?

Reply:
Yes, it works, I have used the gcore to test. And also use it at a real projected ( big server cluster).